src/Api/ShippingLabelFetcher.php

```diff
@@ -57,7 +62,7 @@ public function getLabelContent(): ?string
             return '';
         }
 
-        $this->flashBag->add('success', 'bitbag.ui.shipment_data_has_been_exported'); // Add success notification
+        $this->requestStack->getSession()->getBag('flashes')->add('success', 'bitbag.ui.shipment_data_has_been_exported'); // Add success notification
```

Comment:
I mean to use one variable in all places for better readibility

Reply:
I think there shouldn't be variable at all. There is two separate functions using that `$this->requestStack->getSession()->getBag('flashes')->add()`.

Reply:
It's not the blocker for me, but please read something about Demeter Law :ok_hand: